docs/source/admin/traffic_ops.rst

```diff
@@ -586,6 +590,34 @@ This file sets authentication options for connections to Riak when used as the T
 
 .. impl-detail:: The name of this file is derived from the current database used in the implementation of Traffic Vault - `Riak KV <https://riak.com/products/riak-kv/index.html>`_.
 
+.. _backends.conf:
+
+backends.conf
+"""""""""""""
+This file deals with the configuration parameters of running Traffic Ops as a reverse proxy for certain endpoints that need to be served externally by other backend services. It is a JSON-format set of options and their respective values. `traffic_ops_golang`_ will use whatever file is specified (if any) by its :option:`--backendcfg` option. The keys of the file are described below.
+
+:routes: This is an array of options to configure Traffic Ops to forward requests of specified types to the appropriate backends.
+
+	:path:              The regex matching the endpoint that will be served by the backend, for example, `^api/4.0/foo?$`.
```

Comment:
fyi there's a specific [Sphinx text role for regular expressions](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-regexp) - I think you meant for this to be monospace font e.g. using the `<code>` HTML element, but actually single grave accents (<kbd>\`</kbd>) uses the "default" role which is emphasis, so you're trying to render `^api/4.0/foo?$` but you're getting *^api/4.0/foo?$*

tl;dr: use `:regexp:` for regular expressions, use double grave accents (<kbd>\`</kbd>) for code.